README.md

```diff
@@ -14,6 +14,8 @@ This repository contains the list of communities and job portals you can join an
 <p  align="Left"><a href="DevRel#readme"><b>DevRel</b></a></p>
 <p  align="Left"><a href="Frontend-Development#readme"><b>Frontend-Development</b></a></p>
 <p  align="Left"><a href="Fullstack-Development#readme"><b>Fullstack-Development</b></a></p>
+<p  align="Left"><a href="Mobile-Development#readme"><b>Android-Development</b></a></p>
```

Comment:
```suggestion
```
Android Development is repeated. Please before the 2nd appearance of android development